adapters/amqp-vertx/pom.xml

```diff
@@ -50,8 +50,14 @@
             <configuration>
               <images>
                 <image>
+                  <name>
+                      ${docker.repository}/${project.artifactId}:${project.version}
+                  </name>
                   <build>
                     <from>${java-base-image.name}</from>
+                    <tags>
+                        <tag>${docker.image.additional.tag}</tag>
```

Comment:
IMHO we could add this to the central configuration in the parent POM as well

Reply:
You are right, changed it